### PR TITLE
v5.2.0

### PR DESCRIPTION
**Pull request by brendankenny**
🎉

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,138 @@
...
+* `gather-runner`: treat Chrome interstitials as `pageLoadErrors` ([#9176](https://github.com/googlechrome/lighthouse/pull/9176))
+* `gather-runner`: always reset scroll position ([#9060](https://github.com/googlechrome/lighthouse/pull/9060))
+* `gather-runner`: convert `PAGE_HUNG` to non-fatal `runtimeError` ([#9121](https://github.com/googlechrome/lighthouse/pull/9121))
+* `manifest-parser`: handle blob manifests ([#9088](https://github.com/googlechrome/lighthouse/pull/9088))
+* `page-functions`: don't try to clone a ShadowRoot ([#9079](https://github.com/googlechrome/lighthouse/pull/9079))
+* perf: speed up `tap-target`'s `isVisible()` ([#9056](https://github.com/googlechrome/lighthouse/pull/9056))
+* tracehouse: allow missing FCP ([#9174](https://github.com/googlechrome/lighthouse/pull/9174))
+* tracehouse: move and organize files in `lib/tracehouse/` ([9f80524](https://github.com/googlechrome/lighthouse/commit/9f80524), [1af2edd](https://github.com/googlechrome/lighthouse/commit/1af2edd), [7ba6776](https://github.com/googlechrome/lighthouse/commit/7ba6776), [ddff3d6](https://github.com/googlechrome/lighthouse/commit/ddff3d6))
```
Comment (patrickhulce) [inline changelog.md]: I wonder if this should be called out at all more if folks were relying on the path to do their own trace parsing?

**Comment by connorjclark** [comment]
did we want to start saying the expected chrome release this will be in?

`Stable Chrome release: M77 (Sept 10)`

**Comment by brendankenny** [comment]
added a notable changes section and tried to give fuller descriptions to those and the new audits. Let me know what you think :)

> Stable Chrome release: M77 (Sept 10)

Good idea. I vaguely recall a style guide somewhere about referring to Chrome releases without the M, so reworded slightly. I can change it more, though

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,144 @@
...
+* readme: add `gimbal` integration to related projects ([#9083](https://github.com/googlechrome/lighthouse/pull/9083))
+* readme: update `lighthouse-mocha-example` related project ([#9158](https://github.com/googlechrome/lighthouse/pull/9158))
+* readme: alphabetize related projects ([#9110](https://github.com/googlechrome/lighthouse/pull/9110))
+* fix various typos ([#9411](https://github.com/googlechrome/lighthouse/pull/9411))
+
+## I18n
+
+* introduce script to swap in new locale to LHR ([#8755](https://github.com/googlechrome/lighthouse/pull/8755))
+* enable local debug locale, `en-XL` ([#9192](https://github.com/googlechrome/lighthouse/pull/9192))
+* localize audits in `best-practices` ([#9092](https://github.com/googlechrome/lighthouse/pull/9092))
```
Comment (exterkamp) [inline changelog.md]: So they are prepped for translation, but not translated.  Could add a disclaimer: 

`Note: best-practices audits are not yet translated to all languages in this release. Expected to be translated fully next release.`

Comment (brendankenny) [reply changelog.md]: done (but shorter :)

File: changelog.md (hunk 1 above)
Comment (brendankenny) [reply changelog.md]: > I wonder if this should be called out at all more if folks were relying on the path to do their own trace parsing?

done

**Pull request merged**